apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,221 @@
+                                >
+                                  job.tar.gz
+                                </a>
+                              </td>
+                              <td>
+                                Contains all files necessary for grading; this is what is mounted to
+                                <code>/grade</code> when your job is run.
+                              </td>
+                            </tr>
+                            <tr>
+                              <td>
+                                <a
+                                  href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                                    .grading_job.id}/file/archive.tar.gz"
+                                  >archive.tar.gz</a
+                                >
+                              </td>
+                              <td>
+                                A snapshot of <code>/grade</code> after your job has been executed.
+                              </td>
+                            </tr>`
+                        : ''}
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/results.json"
+                            >results.json</a
+                          >
+                        </td>
+                        <td>
+                          Contains the PrairieLearn-generated results, which includes the contents
+                          of your
+                          <code>results.json</code> as well as some PrairieLearn metadata.
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/output.log"
+                            >output.log</a
+                          >
+                        </td>
+                        <td>
+                          Contains the raw output from stdout/stderr for your job. Lines beginning
+                          with
+                          <code>container &gt;</code> are from your container; the rest are
+                          diagnostic logs from PrairieLearn.
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Job Output</div>
+            <div class="card-body">
+              ${gradingJobQueryResult.grading_job.s3_bucket &&
+              gradingJobQueryResult.grading_job.s3_root_key
+                ? html`
+                    <script>
+                      $(function() => {
```

Comment:
This is invalid syntax, which you can see in the browser console if you open this page:

<img width="325" alt="Screenshot 2023-11-08 at 15 16 56" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/20aad0c4-623c-42f3-8e2f-0732f4855b65">

```suggestion
                      $(() => {
```